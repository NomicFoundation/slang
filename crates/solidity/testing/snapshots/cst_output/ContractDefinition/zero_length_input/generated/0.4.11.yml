# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: ""

Errors: # 1 total
  - >
    Error: Expected '/' or 'a' or 'c'.
       â”€[crates/solidity/testing/snapshots/cst_output/ContractDefinition/zero_length_input/input.sol:0:0]

Tree: null
