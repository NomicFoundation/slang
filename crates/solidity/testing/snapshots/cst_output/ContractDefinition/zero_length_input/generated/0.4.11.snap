# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Error: Expected '/' or 'a' or 'c'.
   â”€[crates/solidity/testing/snapshots/cst_output/ContractDefinition/zero_length_input/input.sol:0:0]

Tree: null
